atlasdb-timelock-server/var/conf/timelock.yml

```diff
@@ -6,3 +6,9 @@ cluster:
   localServer: localhost:8700
   servers:
     - localhost:8700
+
+atomix:
+  security:
```

Comment:
Thanks for the example. I'm a little concerned, though - if you don't have a `truststore.jks` or `keystore.jks` in the module directory, will this prevent the server from starting up? (If it doesn't block that, then this is good.)